components/gfx/display_list/mod.rs

```diff
@@ -206,23 +204,31 @@ impl DisplayList {
     /// inefficient and should only be used for debugging.
     pub fn all_display_items(&self) -> Vec<DisplayItem> {
         let mut result = Vec::new();
+        fn add_all_display_items_for_item(result: &mut Vec<DisplayItem>, item: &DisplayItem) {
```

Comment:
Maybe just call this `flatten`? That's the terminology WR uses for a similar operation.